### ISSUE TITLE
[BUG] Layout Boxed is broken for fixed navbar/footer

### ISSUE THREAD

**Issue by Shidersz** [ColorlibHQ/AdminLTE#3559]
**Describe the bug**
The boxed layout has a lot of visual issues when combined with fixed navbar or footer. Also the slide animation for the right sidebar is wrong. I'm not really sure if this layout is useful for something, but as maintainer of the [Laravel/AdminLTE](https://github.com/jeroennoten/Laravel-AdminLTE) package I'm thinking of dropping the possibility to enable it.

**To Reproduce**
1. Go to https://adminlte.io/themes/v3/pages/layout/boxed.html
2. Open right (control) sidebar and checkout the slide animation (FIRST ISSUE).
3. Open right (control) sidebar and enable fixed navbar (SECOND ISSUE).
4. Open right (control) sidebar and enable fixed footer (THIRD ISSUE).

**Expected behavior**
It should works nicely with fixed navbar or footer.

**Screenshots**

Here you can see how it looks with fixed navbar and footer.

![Screenshot_2021-04-02 AdminLTE 3 Boxed Layout](https://user-images.githubusercontent.com/63609705/113448436-37972680-93d2-11eb-8ac5-aed49f365f84.png)

**Environment (please complete the following information):**
- AdminLTE Version: [e.g. v3.1.0]
- Operating System: Not relevant
- Browser (Version): Not relevant

**Comment by danny007in** [comment]
> Open right (control) sidebar and checkout the slide animation (FIRST ISSUE).
Open right (control) sidebar and enable fixed navbar (SECOND ISSUE).
Open right (control) sidebar and enable fixed footer (THIRD ISSUE).

with image 👍🏼  we can solve issue faster

**Comment by Shidersz** [comment]
@danny007in The above image already shows issues 2 and 3. For the first issue I will need to capture a video, do you have any recommendation tool?

**Comment by danny007in** [comment]
> 
> 
> @danny007in The above image already shows issues 2 and 3. For the first issue I will need to capture a video, do you have any recommendation tool?

it may help u https://github.com/ColorlibHQ/AdminLTE/issues/3558#issuecomment-812770054

**Comment by danny007in** [comment]
in sharex u can record screen

**Comment by REJack** [comment]
Fixed header & footer will not work for boxed layout, `position: fixed;` is only relative to the window not to any other parent.
Control-Sidebar slide bug is fixed with 92ccf78.

**Comment by Shidersz** [comment]
@REJack what if you add the next style that is already applied to `.wrapper`, but to the elements `.main-header` and `.main-footer`. I have give it a fast try and it looks better.

```css
.layout-boxed .wrapper, .layout-boxed .wrapper::before {
    margin: 0 auto;
    max-width: 1250px;
}
```

**Comment by danny007in** [comment]
is this issue resolved

**Comment by Shidersz** [comment]
@danny007in I don't think it is fully addressed, unless you want to specify that fixed navbar and/or footer is not compatible with the boxed layout. I believe it can be possible to make it compatible, but won't be easy since you need to take care of a lots of situations...

**Comment by danny007in** [comment]
> 
> 
> @danny007in I don't think it is fully addressed, unless you want to specify that fixed navbar and/or footer is not compatible with the boxed layout. I believe it can be possible to make it compatible, but won't be easy since you need to take care of a lots of situations...

@REJack Add in Docs about boxed issue

**Comment by REJack** [comment]
> Add in Docs about boxed issue

there is already a note in the docs 😄 
![grafik](https://user-images.githubusercontent.com/1554582/125354646-552a4380-e364-11eb-8080-b4a7b559404d.png)


We will look while the v4 development to get this working.

**Comment by puikinsh** [comment]
Closing as part of issue triage for v4.0.0-rc7. This issue predates the current v4 architecture and is either resolved, outdated, or outside scope. Please open a new issue if the problem persists with the latest release.
